tools/tcpretrans.py

```diff
@@ -325,21 +357,37 @@
 # process event
 def print_ipv4_event(cpu, data, size):
     event = b["ipv4_events"].event(data)
-    print("%-8s %-6d %-2d %-20s %1s> %-20s %s" % (
-        strftime("%H:%M:%S"), event.pid, event.ip,
-        "%s:%d" % (inet_ntop(AF_INET, pack('I', event.saddr)), event.lport),
-        type[event.type],
-        "%s:%s" % (inet_ntop(AF_INET, pack('I', event.daddr)), event.dport),
-        tcpstate[event.state]))
+    if args.sequence:
+        print("%-8s %-6d %-2d %-20s %1s> %-20s %-12s %s" % (
+            strftime("%H:%M:%S"), event.pid, event.ip,
+            "%s:%d" % (inet_ntop(AF_INET, pack('I', event.saddr)), event.lport),
+            type[event.type],
+            "%s:%s" % (inet_ntop(AF_INET, pack('I', event.daddr)), event.dport),
+            tcpstate[event.state], event.seq))
+    else:
+        print("%-8s %-6d %-2d %-20s %1s> %-20s %s" % (
+            strftime("%H:%M:%S"), event.pid, event.ip,
+            "%s:%d" % (inet_ntop(AF_INET, pack('I', event.saddr)), event.lport),
+            type[event.type],
+            "%s:%s" % (inet_ntop(AF_INET, pack('I', event.daddr)), event.dport),
+            tcpstate[event.state]))
 
 def print_ipv6_event(cpu, data, size):
     event = b["ipv6_events"].event(data)
-    print("%-8s %-6d %-2d %-20s %1s> %-20s %s" % (
-        strftime("%H:%M:%S"), event.pid, event.ip,
-        "%s:%d" % (inet_ntop(AF_INET6, event.saddr), event.lport),
-        type[event.type],
-        "%s:%d" % (inet_ntop(AF_INET6, event.daddr), event.dport),
-        tcpstate[event.state]))
+    if args.sequence:
+        print("%-8s %-6d %-2d %-20s %1s> %-20s %-12s %s" % (
+            strftime("%H:%M:%S"), event.pid, event.ip,
+            "%s:%d" % (inet_ntop(AF_INET6, event.saddr), event.lport),
+            type[event.type],
+            "%s:%d" % (inet_ntop(AF_INET6, event.daddr), event.dport),
+            tcpstate[event.state], event.seq))
+    else:
```

Comment:
I'd refactor this so that it's:

```
printf(_first 6 columns_)
if args.sequence:
    printf("%-12s %s" % (tcpstate[event.state], event.seq))
else:
    printf("%s" % (tcpstate[event.state])
```

so we don't have all that duplication